lib/LocalLogisticRegression.js

```diff
@@ -450,7 +450,11 @@ LocalLogisticRegression.prototype.categoryProbability = function (
           index = this.items[fieldId].indexOf(term);
           occurrences = 1;
         }
-        probability += coefficients[shift + index] * occurrences;
+        var lookForIndex = shift + index;
```

Comment:
I don't think I totally understand this change. I guess you had problems when you try to predict with an input data which has an item that has never appeared in the training data. Then the 

``` javascript
index = this.items[fieldId].indexOf(term);
```

gives a negative index. Then I assume we should use the missing coefficient, but that's not the last one in the `coefficients` array... so are you really sure about this change? I mean, it looks like there's a bug in the old code, but I'm not sure it's really solved in your version... I might be mistaken though